style.css

```diff
@@ -4,6 +4,7 @@
   min-height: 100vh;
   flex-direction: column;
   background-color: white;
+  background-image: url('/home/ignatius/Documents/Microverse Projects/Portfolio/images/background-image.png');
```

Comment:
I suggest you fix the path to this background image since its not showing cirrently and your page looks very plain. This is how your page shows currently:
<img width="703" alt="Screenshot 2023-01-14 at 08 40 14" src="https://user-images.githubusercontent.com/48546992/212458547-652adb56-9617-4741-b5fe-35c7d38e7142.png">